.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: node_js
 node_js:
-- '0.10'
-- '0.12'
+- '4'
+- '6'
 # - stable
 sudo: false
+env:
```

Comment:
This junk (and the `addons` below) is needed to get certain Babel modules to properly compile on Node.js 4+.